artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenwireMessage.java

```diff
@@ -30,8 +30,7 @@
 
 import io.netty.buffer.ByteBuf;
 
-// TODO: Implement this
-public class OpenwireMessage implements Message {
+public class OpenWireMessage implements Message {
```

Comment:
you didn't need to change this.. I would keep your changes to a minimal.. only what you really need.

Reply:
ok